www/main/addons/sundar-gutka/components/SundarGutka.jsx

```diff
@@ -83,8 +84,27 @@ const SundarGutka = ({ isShowTranslitSwitch = false, onScreenClose }) => {
             <div className="sttm-loader" />
           ) : (
             <>
-              <header className="navigator-header">{title}</header>
-
+              <header className="navigator-header">
+                {title}
+                <div className="transliterate-eng">
+                  <span>{i18n.t('SETTINGS.ENGLISH_LANGUAGE')} </span>
+                  <div className="switch xs-small">
+                    <input
+                      id="translate-eng"
+                      type="checkbox"
+                      checked={isEngTransliterated}
+                      onChange={() => {
+                        const newState = !isEngTransliterated;
+                        setEngTransliterate(newState);
+                        if (isEngTransliterated) {
+                          onToggle(newState);
```

Comment:
onToggle is not defined

Reply:
Removed, that code wasn't needed anymore.